src/stores/configStore.js

```diff
@@ -408,6 +409,11 @@ export class ConfigStore {
     });
 
     this.selectedLocationNodes.forEach((node) => {
+      // If level is undefined, all is selected and should be filtered out
+      if (node.level === undefined) {
+        return;
+      }
```

Comment:
This fixes the URL parameter issue, but results in no sequences being selected. See below:

![image](https://user-images.githubusercontent.com/3106207/127922762-1b44a049-1f6e-450e-b417-7e01adf8e897.png)

![image](https://user-images.githubusercontent.com/3106207/127922788-f5b05cb0-d08d-4af7-a9c1-1eafee0e7d0f.png)

Refreshing the page results in the default locations (USA, Canada) being selected